scripts/build_dicts.sh

```diff
@@ -1,6 +1,6 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
If this is being changed to `/bin/sh`, the bottom `function refresh_source ()` should be changed to `refresh_source()` (see 2.9.5 Function Definition Command https://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html). I ran into this problem with #1.